incubator/mariadb-cluster/templates/master-deploy.yaml

```diff
@@ -62,6 +62,7 @@ spec:
         {{- end }}
         - name: MARIADB_DATABASE
           value: "{{ .Values.db.name }}"
+        {{- if .Values.replication.enabled }}
```

Comment:
Considering this is a cluster deployment, shouldn't we always enable replication?

Reply:
the idea is to merge this with mariadb-cluster in this repo but later move the changes to the upstream mariadb repo: values.yml would only use a single instance, values-production.yaml would enable replica

Reply:
Considering that the bitnami/mariadb image only applies the configuration on the first run, I think suggest configuring the master with replication support (replication mode/user/password). So that if a user launches the chart with `replication.enabled=false` and later wants to enable replication and add slaves it should work.

Reply:
... otherwise the users would need to have the foresight and launch the chart with `replication.enabled=true,slave.replicas=0` allowing them to add slaves to the cluster when the need arises.

Reply:
Agree, done